docs/freefeatures.rst

```diff
@@ -217,19 +217,6 @@ of using ``pyscript`` is shown below  along with the **examples/arg_printer.py**
     arg 2: 'bar'
     arg 3: 'baz'
 
-.. note::
```

Comment:
I would recommend keeping this note and just updating it to show the required parentheses.